test/python/modules/cfem_diffusion/c_diffusion_2d_1a_linear.py

```diff
@@ -17,7 +17,7 @@
     from pyopensn.solver import DiscreteOrdinatesProblem, SteadyStateSolver
     from pyopensn.fieldfunc import FieldFunctionGridBased
     from pyopensn.fieldfunc import FieldFunctionInterpolationLine, FieldFunctionInterpolationVolume
-    from pyopensn.settings import EnableCaliper
+    from pyopensn.context import EnableCaliper
```

Comment:
@quincy-huhn98 : FYI, since you are moving them to the App. Look at this and advise.